nominees/eduactiv8.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eduActiv8",
+  "description": "eduActiv8 is an open source, free educational program for children 3 to about 10 years old. It helps in learning a range of subjects, but mainly focuses on maths",
+  "website": "https://www.eduactiv8.org/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/imiolek-ireneusz/eduActiv8/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceURL": "https://www.eduactiv8.org/about/"
```

Comment:
In as much as possible, try to have evidenceURL point to articles, blogs etc that help show how the project is relevant to the SDG in question. @vaishnavi-ch it would be good to replace the existing link with this [article](https://opensource.com/article/19/4/early-literacy-tools) or a stronger validation you may find. You may even add an `evidenceText` and borrow the relevant text from that link shared that talks about educActiv8. Thanks.